Sources/xcodeproj/Objects/Project/PBXProjEncoder.swift

```diff
@@ -321,6 +321,7 @@ final class PBXProjEncoder {
             let element = arg as! PBXProjElement
             var array = [String]()
             var tmpStateHolder = StateHolder()
+            tmpStateHolder.indent = 2
```

Comment:
`tmpStateHolder.indent = stateHolder.indent` guess it will be the same as `tmpStateHolder.indent = 2`.
So it was originally, but then for some reason I decided that it was unnecessary.